www/components/SPA/ModulePage.tsx

```diff
@@ -22,7 +20,7 @@ export default function ModulePage() {
             {mod ? (
               <Module pkg={pkg} mod={mod} />
             ) : (
-              `Module ${modName} not found 🤪`
+              `Module "${modName || ""}" not found 🤪`
```

Comment:
can `modName` be null/undefined here? same question further down below

Reply:
In principle, maaaybe? The way this component is used currently (i.e. in exactly one React router `Route` where these params are set), no, in practice it can't.

Reply:
(but we need to handle the case that it somehow magically *is*, and TypeScript is correct in pointing this out)

Reply:
Can't we just declare it to be non-null and get a type error when we inevitably try and pass in null or undefined in the future? No big deal but imo there's a real difference between "I don't even know what I'm looking for but it ain't there" and "this thing is not there"